guides/common/modules/ref_capsule-storage-requirements.adoc

```diff
@@ -22,18 +22,3 @@ endif::[]
 |/usr |3 GB |Not Applicable
```

Comment:
Should the EL8 header on line 12 be dropped to simplify the page structure?

Reply:
Addressed :+1: